controllers/aliens.js

```diff
@@ -0,0 +1,73 @@
+const Alien = require('../models/alien');
+
+const getAllAliens = async (req, res) => {
+  try {
+    const aliens = await Alien.find({});
+    res.status(200).json({ aliens });
+  } catch (error) {
+    res.status(500).json({ msg: error });
+  }
+};
+
+const createAlien = async (req, res) => {
+  try {
+    const alien = await Alien.create(req.body);
+    res.status(201).json({ alien });
+  } catch (error) {
+    res.status(500).json({ msg: error });
+  }
+};
+
+const getAlien = async (req, res) => {
+  try {
+    const { id: alienID } = req.params;
+    const alien = await Alien.findOne({ _id: alienID });
+    if (alien) {
+      res.status(201).json({ alien });
```

Comment:
Hi @tsheporamantso 
status code for `get`, `patch` and `delete` is 200, kindly change the code on the routes mentioned.

Reply:
Hi Reviewer,

Noted and thank you so much I will make changes as requested.
<img src="https://media3.giphy.com/media/g1GUaKrhsdI6syOfwR/giphy.gif"/>